src/components/ContentSidebar/ActivityFeed/comment/Comment.scss

```diff
@@ -119,10 +119,6 @@
 
         &.bcs-is-pending {
             opacity: .4;
-
-            .bcs-comment-text {
-                margin-bottom: 25px;
```

Comment:
Was this from BRUI or did we add this?

Reply:
Looks like it was added during the dummy data prototype commit